src/index.js

```diff
@@ -126,6 +128,28 @@ Object.assign(
 )
 
 /*
+ * This creates a glamorousComponentFactory for every jsxstyle element so you
+ * can simply do:
+ * const GreenBlock = glamorous.block({
+ *   backgroundColor: 'green',
+ *   padding: 20,
+ * })
+ * <GreenBlock>Click Me!</GreenBlock>
+ */
+Object.assign(
+  glamorous,
+  jsxStyleComponentTags.reduce(
+    (getters, tag) => {
+      const capitalTag = capitalize(tag)
```

Comment:
What I mean is, in the code block example above it shows: `glamorous.block`, but here it's capitalizing the tag, so what you'll actually have to do is: `glamorous.Block`, which will actually get overwritten in the next `Object.assign` anyway.

In any case, I'm unsure I want to expose this. And if we do I think I'd want to expose it on `glamorous.display` as discussed in the other comment.